src/openrct2/world/Map.cpp

```diff
@@ -2393,8 +2402,8 @@ void game_command_place_large_scenery(
 
         if (!gCheatsDisableClearanceChecks
             && !map_can_construct_with_clear_at(
-                   curTile.x, curTile.y, zLow, zHigh, &map_place_scenery_clear_func, bl, flags, &supportsCost,
-                   CREATE_CROSSING_MODE_NONE))
+                curTile.x, curTile.y, zLow, zHigh, &map_place_scenery_clear_func, bl, flags, &supportsCost,
+                CREATE_CROSSING_MODE_NONE))
```

Comment:
Clang-format wants the indention here restored.

If your clang-format formatted it this way... I noticed a similar issue earlier. The version on Travis probably doesn't match your local machine.

Reply:
Yes it's quite annoying. I seem to be hitting a few clang format bugs.

Reply:
Might be better to make it a new nested if block.